bin/rucio

```diff
@@ -1459,10 +1460,16 @@ def move_rule(args):
     Update a rule.
     """
     client = get_client(args)
+
+    override = {}
+    if args.activity:
+        override['activity'] = None if args.activity.lower() == "none" else args.activity
+    if args.source_replica_expression:
+        override['source_replica_expression'] = None if args.source_replica_expression.lower() == "none" else args.source_replica_expression
```

Comment:
How should we document the `$ rucio move-rule --source-replica-expression None RULE_ID RSE_EXPRESSION` case? In the other cases, it's not documented at all.

Reply:
May I suggest adding ‘Use 'None' to remove' or something similar here?
https://github.com/rucio/rucio/blob/4a896696917bee18b49c902b289d189ede697dfc/bin/rucio#L2488